Makefile

```diff
@@ -34,6 +34,9 @@ dev:
 utop: all
 	OCAMLPATH=_build/install/default/lib:$(OCAMLPATH) utop
 
+ocamlformat:
+	dune build @fmt
```

Comment:
```suggestion
	dune build @fmt --auto-promote
```

We could combine two steps: formatting and promoting the changes into one command. I don't see a problem with that given that we keep files in git. If you'd like things to be even safer, then we could add a check that there are no uncommitted changes before executing this command.